### ISSUE TITLE
Run test suite against WordPress nightly

### ISSUE THREAD

**Issue by danielbachhuber** [fusioneng/speed-bumps#87]
Doing so can help catch issues like #86

**Comment by goldenapples** [comment]
Good point. I'm not sure why we haven't done that yet (other than just not updating the plugin lately).

In addition, we should add PHP 7 to the test matrix; and we can probably remove PHP 5.3.
